line-sdk/src/test/java/com/linecorp/linesdk/auth/internal/BrowserAuthenticationApiTest.java

```diff
@@ -200,8 +203,8 @@ public void testConvertToCompatibleIntent() throws Exception {
         assertNull(intentHolder.getStartActivityOptions());
 
         // Test for non new LINE and single target
-        setLineAppVersion("6.8.0");
```

Comment:
Why do we need to remove this line? The comment above says we need to test `new LINE` and `non new LINE` test cases?

Reply:
Now, we do not exclude LINE version below 6.9.0 any more. I will check how to set it to null.